Gopkg.toml

```diff
@@ -130,3 +130,7 @@ ignored = [
 [[constraint]]
   branch = "master"
   name = "github.com/kube-object-storage/lib-bucket-provisioner"
+
+[[override]]
+  name = "github.com/openshift/machine-api-operator"
```

Comment:
Use this instead: https://github.com/kubevirt/machine-remediation-operator/blob/master/pkg/apis/machineremediation/v1alpha1/machinedisruptionbudget_types.go


Also, why are we trying to override this to a particular revision?

Reply:
using this `github.com/openshift/machine-api-operator` because openshift will comming up with the machine-api-operator.

Reply:
we are overriding the particular version because the MDB struct is not available in the last release.